app/view/form/PrintController.js

```diff
@@ -576,5 +577,21 @@ Ext.define('MoMo.client.view.form.PrintController', {
         if (field.getName() !== field.childFieldKey) {
             attributes[field.childFieldKey] = field.getValue();
         }
+    },
+
+    saveLoadSettings: function() {
+        var me = this;
+        var viewModel = me.getView().getViewModel();
+        Ext.create('Ext.window.Window', {
+            width: 800,
+            height: 350,
+            modal: true,
+            bind: {
+                title: viewModel.get('saveLoadSettings')
```

Comment:
Or just `{saveLoadSettings}`?

Reply:
tried it, didnt work, will keep it this way